### DISCUSSION TITLE
setup multiple bridges with emqx broker

### DISCUSSION THREAD

**Discussion by 4ff4n** [emqx/emqx#12982] [General]
Hey there! I'm looking to configure two bridges with my EMQX broker to send messages to two separate devices registered on an IoT hub. I've attempted the setup outlined below, but it doesn't seem to be functioning as expected.
## Configuration for Bridge 1
bridge.mqtt.aws1.address = {address}:8883
bridge.mqtt.aws1.proto_ver = mqttv4
bridge.mqtt.aws1.start_type = auto
bridge.mqtt.aws1.clientid = sensor1
bridge.mqtt.aws1.clean_start = true
bridge.mqtt.aws1.username ={address}/sensor1
bridge.mqtt.aws1.password = {pass}
bridge.mqtt.aws1.forwards = devices/sensor1/messages/events/#
bridge.mqtt.aws1.forward_mountpoint = bridge/aws/${node}/
bridge.mqtt.aws1.ssl = on
bridge.mqtt.aws1.cacertfile = {path to ca cert}
bridge.mqtt.aws1.certfile = /etc/emqx/certs/client-cert.pem
bridge.mqtt.aws1.keyfile = /etc/emqx/certs/client-key.pem
bridge.mqtt.aws1.keepalive = 60s
bridge.mqtt.aws1.tls_versions = tlsv1.3,tlsv1.2,tlsv1.1,tlsv1
bridge.mqtt.aws1.reconnect_interval = 30s
bridge.mqtt.aws1.retry_interval = 20s
bridge.mqtt.aws1.batch_size = 32
bridge.mqtt.aws1.max_inflight_size = 32
bridge.mqtt.aws1.queue.replayq_dir = /var/lib/emqx/replayq/emqx_aws_bridge/
bridge.mqtt.aws1.queue.replayq_seg_bytes = 10MB
bridge.mqtt.aws1.queue.max_total_size = 5GB

Configuration for Bridge 2

bridge.mqtt.aws2.address = example.com:8883
bridge.mqtt.aws2.proto_ver = mqttv5
bridge.mqtt.aws2.start_type = auto
Add other configuration parameters for Bridge 2...
.............
Looking forward to your kind assistance
BR,
Affan

**Comment by kjellwinblad** [comment]
Hi,

It looks to me that the configuration paths are incorrect. What errors do you get when starting EMQX with those configurations? It would also be interesting to know what version of EMQX you are using? I guess you already know that one can configure connectors/actions (called bridges in earler versions of EMQX) via the EMQX dashboard. If you do that you can look at the resulting configuration file to see the configuration paths for the different settings. Hope this helps!

Best regards,
Kjell
